docs/.vuepress/config.ts

```diff
@@ -108,6 +108,7 @@ export default defineUserConfig<DefaultThemeOptions, ViteBundlerOptions>({
               '/components/label',
               '/components/menu',
               '/components/modal',
+              '/components/kmodalfullscreen',
```

Comment:
```suggestion
              '/components/modal-fullscreen',
```
issue: wrong file name

Reply:
Fixed